CDCgov/prime-reportstream
# 9080 - STLT Report Details Design

This PR ...

As part of the STLT Dashboard, we need to implement the design of the report details page. This implementation is strictly visual, and we'll have a _separate ticket for the table and hooking it up to the API afterward_.

Test Steps:
1. Navigate to RS
2. Select a Receiver
3. Go to /data-dashboard
4. Click on a Report ID
5. Visually should match [Figma](https://www.figma.com/file/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?type=design&node-id=2369-27816&t=sSRJvSYgCUQbL2TO-0)
6. Also, Updates the button on the Report Details of the Daily Data page. The button will show/hide based off of the expiration date

## Changes
<img width="1764" alt="Screenshot 2023-05-11 at 9 48 38 AM" src="https://github.com/CDCgov/prime-reportstream/assets/102491809/f4cefbb7-adf2-42cb-9e33-08417a78a3d7">

<img width="1752" alt="Screenshot 2023-05-11 at 9 19 34 AM" src="https://github.com/CDCgov/prime-reportstream/assets/102491809/5972186b-f87a-4ce2-8bc1-c0976968f237">

Daily Data
<img width="1452" alt="Screenshot 2023-05-11 at 12 24 26 PM" src="https://github.com/CDCgov/prime-reportstream/assets/102491809/ee340c57-4a00-46ad-b961-ff156a49af9d">

## Checklist

### Testing


## Linked Issues
- Fixes #9080 

## To Be Done
*Table will be using new table component
- #9474
